crates/ruff_python_ast/src/nodes.rs

```diff
@@ -2983,9 +3012,9 @@ mod size_assertions {
     use super::*;
     use static_assertions::assert_eq_size;
 
-    assert_eq_size!(Stmt, [u8; 176]);
-    assert_eq_size!(StmtFunctionDef, [u8; 128]);
-    assert_eq_size!(StmtClassDef, [u8; 168]);
+    assert_eq_size!(Stmt, [u8; 184]);
+    assert_eq_size!(StmtFunctionDef, [u8; 136]);
+    assert_eq_size!(StmtClassDef, [u8; 176]);
```

Comment:
I will look into boxing in a separate PR (for type parameters and perhaps arguments).